src/types/accessibilityFulfillment.ts

```
@@ -0,0 +1,105 @@
+// ─── Real-Time Accessibility Need Fulfillment Tracker Types ──────────────────────
+
+import { GeoPoint } from "./accessibility";
+
+export type AccommodationCategory =
+  | "mobility"
+  | "auditory"
+  | "visual"
+  | "cognitive"
+  | "spatial";
```

ESLint

[error] Replace `⏎··|·"mobility"⏎··|·"auditory"⏎··|·"visual"⏎··|·"cognitive"⏎·` with `·"mobility"·|·"auditory"·|·"visual"·|·"cognitive"`

(prettier/prettier)